R/resolver.R

```diff
@@ -311,7 +311,7 @@ translate_legacy_resolver_expr <- function(quo, null_means) {
 resolve_rows_l <- function(expr, data) {
 
   if (is_gt(data)) {
-    row_names <- dt_stub_df_get(data)$rowname
+    row_names <- dt_stub_df_get(data)$row_id
```

Comment:
Does this change imply breaking backwards compatibility for users of the library?

Reply:
I looked at this pretty closely and I don’t think it does. It adds better targeting of rows and doesn’t take anything away.

Reply:
We did a similar thing awhile back where we upgrading spanner column labels to have and ID and a label (previously you had to use a label for targeting) and this takes a similar approach.

Reply:
That sounds good; I just wanted to make sure.